source/_patterns/02-organisms/08-header/03-lend-menu.mustache

```diff
@@ -33,12 +33,12 @@
 	                <a href="#" data-kv-accordion aria-controls="lend-menu-personalized-panel" aria-expanded="false" class="section-title">My Kiva<svg class="icon icon-small-chevron-mobile"><use xlink:href="#icon-small-chevron-mobile" /></svg></a>
 	                <ul id="lend-menu-personalized-panel" aria-hidden="true">
 						{{#has_favorites}}
-							<li class="favorites-href-in-nav"><a kvtrackevent="TopNav|click-Lend-Favorites" href="{{favorites_href}}">Starred loans</a></li>
-							<li class="no-favorites-href-in-nav hide"><a kvtrackevent="TopNav|click-Lend-Favorites" href="#" class="disabled">Starred loans</a></li>
+							<li class="favorites-href-in-nav"><a kvtrackevent="TopNav|click-Lend-Favorites" href="{{favorites_href}}">Saved loans</a></li>
+							<li class="no-favorites-href-in-nav hide"><a kvtrackevent="TopNav|click-Lend-Favorites" href="#" class="disabled">Saved loans</a></li>
 						{{/has_favorites}}
 						{{^has_favorites}}
-							<li class="no-favorites-href-in-nav"><a kvtrackevent="TopNav|click-Lend-Favorites" href="#" class="disabled">Starred loans</a></li>
-							<li class="favorites-href-in-nav hide"><a kvtrackevent="TopNav|click-Lend-Favorites" href="{{favorites_href}}">Starred loans</a></li>
+							<li class="no-favorites-href-in-nav"><a kvtrackevent="TopNav|click-Lend-Favorites" href="#" class="disabled">Saved loans</a></li>
+							<li class="favorites-href-in-nav hide"><a kvtrackevent="TopNav|click-Lend-Favorites" href="{{favorites_href}}">Saved loans</a></li>
```

Comment:
For a future optimization, I'm pretty sure you could have only a single entry here and use the template conditions to apply classes when needed...

Reply:
This could certainly be refactored. I went with the lightest touch possible and just updated the text displayed. Starred -> Saved.